### PR TITLE
autotest: provide capability to pass boards as argument

### PR DESCRIPTION
**Pull request by authmillenon**
The first commit is optional, the last a little bit more critical.

### CODE REVIEW

File: dist/tools/autotest/run.py [hunk 1]
```diff
@@ -13,20 +13,21 @@
 BOARDS = [autotest.Board("native")]
```
Comment (cgundogan) [inline dist/tools/autotest/run.py]: this line can be removed then?

File: dist/tools/autotest/autotest/utils.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim:fenc=utf-8
+#
+# Copyright © 2016 Martine Lenders <mail@martine-lenders.eu>
+#
+# Distributed under terms of the MIT license.
+
+class Borg(object):
```
Comment (cgundogan) [inline dist/tools/autotest/autotest/utils.py]: love the name (:

File: dist/tools/autotest/run.py (hunk 1 above)
Comment (authmillenon) [inline dist/tools/autotest/run.py]: Can be, yes :-) (will do tomorrow).

File: dist/tools/autotest/autotest/utils.py (hunk 2 above)
Comment (authmillenon) [inline dist/tools/autotest/autotest/utils.py]: I have to credit [David Asher](http://code.activestate.com/recipes/users/70683/) for that name ;-)

**Pull request merged**